src/main/kotlin/icons/Icons.kt

```diff
@@ -0,0 +1,19 @@
+package icons
+
+import com.intellij.ui.IconManager
+import javax.swing.Icon
+
+class Icons {
+
+    @JvmField
+    public val terminal = loadIcon("icons/terminal.svg", (hashCode() + "terminal".hashCode()).toLong(), 2)
```

Comment:
I don't understand the purpose of this class ? As it's new, where it's been used in the code ?

Reply:
oh, was added by accident. Have to remove it.